JSON/requirements-sample-fio.json

```diff
@@ -14,12 +14,15 @@
         ]
     },
     "validations": {
-        "size_KMG": {
-            "description": "bytes in k/K (1024), m/M (1024^2) or g/G (1024^3): 4k 16M 1g",
+        "size_BKMG": {
+            "description": "bytes in B/K/M/G, converted to K, shown in KB",
             "args": [ "bs" ],
-            "vals": "[0-9]+[kbmgKBMG]",
-            "transform": [ "s/([0-9]+)[gG]/($1*1024).\"M\"/e",
-                           "s/([0-9]+)[mM]/($1*1024).\"K\"/e" ]
+            "vals": "[0-9]+[BKMG]?",
+            "convert": "",
+            "transform": {
+                "search": "[^0123456789\\.]",
```

Comment:
Are you sure you don't mean `"search": "[^[1-9][0-9]*\\.]"`?  As written I think this is exclusively looking for the string `0123456789`.

Reply:
It doesn't matter, this one is not to validate leading zeros. I can change it to make it consistent, though.